webpack.dev.config.js

```diff
@@ -88,13 +89,19 @@ Object.assign(config, {
       filename: '[name].css',
     }),
     new webpack.ProvidePlugin({
-      Backgrid: path.resolve(path.join(__dirname, 'openassessment/xblock/static/js/lib/backgrid/backgrid')),
+      Backgrid: path.resolve(path.join(__dirname, 'openassessment/xblock/static/js/lib/backgrid/backgrid'))
+    }),
+    new WebpackManifestPlugin({
+      seed: {
+        is_dev_server: process.env.WEBPACK_DEV_SERVER || false,
+        base_url: process.env.WEBPACK_DEV_SERVER ? `http://localhost:${config.devServer.port}/`: '/static/dist/',
+      },
+      writeToFileEmit: true
     }),
+    new webpack.HotModuleReplacementPlugin(),
   ],
 });
 
-config.resolve.modules = ['node_modules'].concat(
-  path.resolve(__dirname, 'openassessment/xblock/static/js/src'),
-);
+config.resolve.modules = ['node_modules', path.resolve(__dirname, 'openassessment/xblock/static/js/src')]
```

Comment:
eslint errors in this file (missing commas/semicolons, and wrong quotes). not all of these complaints are new w/ this change.